mindjunkies/lecture/tests/test_views.py

```
@@ -97,28 +99,59 @@ def test_lecture_video_forbidden_for_non_enrolled(
 
 
 @pytest.mark.django_db
-def test_lecture_pdf_view(client, user, lecture, lecture_pdf):
+def test_create_video_content_success(client):
+    # Create a user and log them in
+    user = baker.make(User)
```

Ruff (0.8.2)

Undefined name `User`

(F821)